pkg/snapshot/snapshot.sh

```diff
@@ -0,0 +1,31 @@
+TYK_MODE=pro
+TYK_URL=http://host.minikube.internal:3000
+TYK_AUTH="760c51da47624c5c501eecd282a7951f"
+TYK_ORG="5e9d9544a1dcd60001d0ed20"
+OUTPUT_DIR="$HOME/snapshot/data"
+CATEGORY=""
+NAMESPACE="default"
+
+minikube start --mount-string="$OUTPUT_DIR:/data" --mount
+minikube addons enabled ingress
+
+helm upgrade "cert-manager" jetstack/cert-manager \
+  --install \
+  --version v1.10.1 \
+  --set "installCRDs=true" \
+  --set "prometheus.enabled=false" \
+  --namespace "$NAMESPACE"
+  --wait
+
+kubectl create secret -n "$NAMESPACE" generic tyk-operator-conf \
+  --from-literal "TYK_MODE=$TYK_MODE" \
+  --from-literal "TYK_URL=$TYK_URL" \
+  --from-literal "TYK_AUTH=$TYK_AUTH" \
+  --from-literal "TYK_ORG=$TYK_ORG" \
+  --dry-run=client -o yaml | kubectl apply -f -
+
+sed "s/CATEGORY/$CATEGORY/g" snapshot.job.template.yaml | \
+  kubectl delete -n "$NAMESPACE" -f -
+
+sed "s/CATEGORY/$CATEGORY/g" snapshot.job.template.yaml | \
+  kubectl apply -n "$NAMESPACE" -f -
```

Comment:
It's worth looking at Makefile in the root of the project. This is where operator team put helper functions.

Reply:
It would but I am not sure this minikube approach is the right one. I will give it another try later.